src/sentry/mediators/sentry_app_installation_tokens/creator.py

```diff
@@ -43,7 +43,7 @@ def audit(self):
             create_audit_entry(
                 request=self.request,
                 organization=self.organization,
-                target_object=self.sentry_app_installation_token.id,
+                target_object=self.api_token.id,
```

Comment:
@MeredithAnya Good catch, we shouldn't expose the `sentry_app_installation_token` to the outside world.